docs/usage/quickstart.rst

```diff
@@ -217,25 +238,34 @@ Similarly, we can list all deployed Helm_ applications::
 
 Cluster Services
 ----------------
-Various services to operate and monitor your MetalK8s cluster are provided. To
-access these, first create a secure tunnel into your cluster by running
-``kubectl proxy``. Then, while the tunnel is up and running, the following
-tools are available:
-
-+-------------------------+---------------------------------------------------------+-------------------------------------------------------------------------------------------------+
-| Service                 | Role                                                    | Link                                                                                            |
-+=========================+=========================================================+=================================================================================================+
-| `Kubernetes dashboard`_ | A general purpose, web-based UI for Kubernetes clusters | http://localhost:8001/api/v1/namespaces/kube-system/services/https:kubernetes-dashboard:/proxy/ |
-+-------------------------+---------------------------------------------------------+-------------------------------------------------------------------------------------------------+
-| `Grafana`_              | Monitoring dashboards for cluster services              | http://localhost:8001/api/v1/namespaces/kube-ops/services/kube-prometheus-grafana:http/proxy/   |
-+-------------------------+---------------------------------------------------------+-------------------------------------------------------------------------------------------------+
-| `Cerebro`_              | An administration and monitoring console for            | http://localhost:8001/api/v1/namespaces/kube-ops/services/cerebro:http/proxy/                   |
-|                         | Elasticsearch clusters                                  |                                                                                                 |
-+-------------------------+---------------------------------------------------------+-------------------------------------------------------------------------------------------------+
-| `Kibana`_               | A search console for logs indexed in Elasticsearch      | http://localhost:8001/api/v1/namespaces/kube-ops/services/http:kibana:/proxy/                   |
-+-------------------------+---------------------------------------------------------+-------------------------------------------------------------------------------------------------+
-
-See :doc:`../architecture/cluster-services` for more information about these
+Services to operate and monitor your MetalK8s cluster are provided. To
+access these dashboards:
+
+1. Copy the credentials in
```

Comment:
If you must deploy from the remote machine in the first place instead of using your local machine as a controller, chances are high you can't reach the K8s master node(s), so this won't work anyway.

If you deployed from the local machine, there's no need to copy whatever files around.